src/components/charts/galaxySelector/Points.jsx

```diff
@@ -20,16 +20,18 @@ class Points extends React.PureComponent {
     return (
       <g className={`data-points ${pointClasses}`}>
         {data.map(d => {
-          const { id, color } = d;
+          const { id, color, radius } = d;
+          const xVal = d[xValueAccessor];
+          const modR = 0.6 * radius;
           const isSelected = !!find(selectedData, { id: d.id });
           const isActive = active ? active.id === d.id : false;
-
           return (
             <Point
               key={`point-${id}`}
-              {...{ id, isSelected, isActive }}
+              {...{ id, isActive, isSelected }}
+              radius={xScale(xVal - modR) - xScale(xVal + modR)}
               classes={pointClasses}
-              x={xScale(d[xValueAccessor])}
+              x={xScale(xVal)}
               y={yScale(d[yValueAccessor])}
```

Comment:
why not do the same with the `yValueAccessor` ?

Reply:
no reason I guess. As I rule of thumb I just only assign a value to a variable if it gets used more than once.  Force of habit more than anything I spoze.